README.md

```diff
@@ -998,6 +998,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 
 **[`^        back to top        ^`](#)**
 
+- [2FAuth](https://github.com/Bubka/2FAuth) - A web app to manage your Two-Factor Authentication (2FA) accounts and generate their security codes. ([Demo](https://demo.2fauth.app/)) `AGPL-3.0` `PHP`
```

Comment:
Wouldn't this be a better fit in the Password Managers category?

Reply:
This was my first thought, but all items of the PM category are strictly password managers unlike 2FAuth.

Both relates to the same purpose, manage security artifacts of accounts. Therefore shouldn't the category be renamed to something a little less restrictive ? i.e _Account & Password managers_ ?

Reply:
I think it should rather go to https://github.com/n1trux/awesome-sysadmin#identity-management

Reply:
But 2FAuth is not a sysadmin tool, it's exactly a selfhosted app and it is why I started to develop it, to have a selfhosted solution for a daily end-user need, the 2FA token generation.

The target audience is clearly the one of the awesome-selfhosted page in my opinion.